### PR TITLE
Add AWS docs

### PR DESCRIPTION
**Pull request by flavio**
**WARNING: DO NOT MERGE**  - wait for the green light from either @flavio or @jordimassaguerpla - these docs reference things that are not yet shipped to our customers.

# What this is about

Extend the AWS docs to reflect the latest changes. Re-enable them into the official docs.

# Final warning
I wasn't able to render the **whole** docs on my computer, I hope the cross-doc links are working as expected.

### CODE REVIEW

**Comment by r0ckarong** [comment]
Blocks #698 698

**Comment by flavio** [comment]
I've rebased the PR to fix the conflicts. We need help reviewing these contents.

ping @jordimassaguerpla @r0ckarong @nkoranova

**Comment by nkoranova** [comment]
Thanks for the PR, I'll have a look now.

File: adoc/deployment-aws.adoc [hunk 1]
```diff
@@ -1,98 +1,208 @@
...
+=== AWS Deployment
+
+The AWS deployment created by our {tf} template files leads to the
+creation of the infrastructure described in the next paragraphs.
+
+==== Network
+
+All the infrastructure is created inside of a user specified AWS region.
+All the resources are currently located inside of the same availability
+region.
```
Comment (innobead) [inline adoc/deployment-aws.adoc]:
```suggestion
zone.
```

**Comment by nkoranova** [comment]
Has been checked by @kravciak who also tested it in October. He says it's ok, so this is IMHO ready to be merged!

**Comment by r0ckarong** [comment]
Added some minor formatting. The rest LGTM.

**Comment by flavio** [comment]
woot, thanks for merging the PR @r0ckarong and @nkoranova ! The whole document looks great on susedoc.github.io :tada: :champagne:

**Pull request merged**